dask/array/core.py

```diff
@@ -2231,7 +2231,14 @@ def astype(self, dtype, **kwargs):
             By default, astype always returns a newly allocated array. If this
             is set to False and the `dtype` requirement is satisfied, the input
             array is returned instead of a copy.
+
+            .. note::
+
+                Dask does not respect the contiguous memory layout of the array,
+                and will ignore the ``order`` keyword argument.
+                The default order is 'C' contiguous.
         """
+        kwargs.pop("order", None)  # `order` is not respected, so we remove this kwarg
```

Comment:
Sorry I didn't catch this sooner, but does it make sense to raise an error or a warning in this case? I think the only reason why we might not want to do that is if the user doesn't control the kwargs.

Reply:
Ah I read the PR description better and I think I get why we don't want to raise any error or warning in this case. Feel free to ignore.